--- rapidsvn-0.6.0/configure.in	2004-05-24 18:17:09.000000000 +0200
+++ rapidsvn-0.6.0/configure.in.new	2004-09-17 03:12:12.478412688 +0200
@@ -7,8 +7,8 @@
 AC_PROG_LIBTOOL
 
 abs_srcdir="`cd $srcdir && pwd`"
-CPPFLAGS="$CPPFLAGS -Wall -g -fexceptions -I$abs_srcdir/include"
-CXXFLAGS="$CXXFLAGS -Wall -g -fexceptions"
+CPPFLAGS="$CPPFLAGS -Wall -fexceptions -I$abs_srcdir/include"
+CXXFLAGS="$CXXFLAGS -Wall -fexceptions"
 
 dnl
 dnl APR
