--- rapidsvn-0.4.0/src/svncpp/revision.hpp.orig	2003-12-06 21:42:26.000000000 +0000
+++ rapidsvn-0.4.0/src/svncpp/revision.hpp	2004-01-09 23:07:50.000000000 +0000
@@ -62,8 +62,9 @@
      * Constructor
      *
      * @param date 
+     * @param dummy to distinguish from svn_revnum_t on 64-bit machines
      */
-    Revision (const apr_time_t date);
+    Revision (const apr_time_t date, int dummyitsdate);
 
     /**
      * Copy constructor
--- rapidsvn-0.4.0/src/svncpp/revision.cpp.orig	2003-12-06 21:42:26.000000000 +0000
+++ rapidsvn-0.4.0/src/svncpp/revision.cpp	2004-01-09 23:08:33.000000000 +0000
@@ -35,7 +35,7 @@
     m_revision.value.number = 0;
   }
 
-  Revision::Revision (const apr_time_t date)
+  Revision::Revision (const apr_time_t date, int dummyitsdate)
   {
     m_revision.kind = svn_opt_revision_date;
     m_revision.value.date = date;
--- rapidsvn-0.4.0/src/filelist_ctrl.hpp.orig	2003-12-06 21:42:21.000000000 +0000
+++ rapidsvn-0.4.0/src/filelist_ctrl.hpp	2004-01-09 23:39:40.000000000 +0000
@@ -24,8 +24,8 @@
 /**
  * Define an array of indexes of the selected items.
  */
-#ifdef WX_DEFINE_ARRAY_INT
-WX_DEFINE_ARRAY_INT (long, IndexArray);
+#ifdef WX_DEFINE_ARRAY_LONG
+WX_DEFINE_ARRAY_LONG (long, IndexArray);
 #else
 WX_DEFINE_ARRAY (long, IndexArray);
 #endif
